normandy/recipes/models.py

```diff
@@ -21,13 +20,48 @@ class Locale(models.Model):
     code = models.CharField(max_length=255, unique=True)
     english_name = models.CharField(max_length=255, blank=True)
     native_name = models.CharField(max_length=255, blank=True)
+    order = models.IntegerField(default=100)
 
     class Meta:
-        ordering = ['code']
+        ordering = ['order', 'code']
 
     def __str__(self):
         return '{self.code} ({self.english_name})'.format(self=self)
 
+    def matches(self, other):
+        return self.code.lower() == other.lower()
+
+
+class ReleaseChannel(models.Model):
+    """Release channel of Firefox"""
+    slug = models.SlugField(max_length=255, unique=True)
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ['slug']
+
+    def __str__(self):
+        return self.name
+
+    def matches(self, other):
```

Comment:
I'm going to change this with the other matching refactors we talked about, since changing this sort of breaks `check_many.